StortSpelProjekt/Game/src/main.cpp

```diff
@@ -231,6 +231,7 @@ Scene* GameScene(SceneManager* sm)
 	AudioBuffer* music = al->LoadAudio(L"../Vendor/Resources/Audio/backgroundMusic.wav", L"Music");
 	music->SetAudioLoop(0);
 
+    AudioBuffer* explosionSound = al->LoadAudio(L"../Vendor/Resources/Audio/Explosion.wav", L"ExplosionSound");
```

Comment:
Indentation is not consistent. I know this is a github thing with how it displays tabs vs spaces, but we want things to look neat and tidy, right :>

Reply:
Can't do a thing about it.